rest/route/middleware.go

```diff
@@ -330,28 +330,19 @@ func (m *podAuthMiddleware) ServeHTTP(rw http.ResponseWriter, r *http.Request, n
 	id := gimlet.GetVars(r)["pod_id"]
 	if id == "" {
 		if id = r.Header.Get(evergreen.PodHeader); id == "" {
-			gimlet.WriteResponse(rw, gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
-				StatusCode: http.StatusUnauthorized,
-				Message:    "missing pod ID",
-			}))
+			gimlet.WriteResponse(rw, gimlet.MakeJSONErrorResponder(errors.New("missing pod ID")))
```

Comment:
maybe these should return a 400?

Reply:
`MakeJSONErrorResponder` returns 400 bad request by default unless you give it a `gimlet.ErrorResponder` as an error.